portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteApps.path

```diff
@@ -25,6 +25,11 @@
 	<td>//div[contains(@class,'sidebar-body__add-panel')]//div[contains(@class, 'text-truncate title')][contains(text(),'${key_remoteAppName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REMOTE_APP_PROPERTY_HTML</td>
+	<td>//vanilla-counter[contains(@test-data-user,'QAuser')]</td>
```

Comment:
I suggest using a variable here instead of hardcoding `QAuser`. Like `${key_customElementProperty}`